web/.env.development

```diff
@@ -13,19 +13,18 @@ REACT_APP_MODELS_API=${API_URL}
 REACT_APP_SEARCH_API=${API_URL}
 REACT_APP_TAXONOMIES_API=${API_URL}
 
-REACT_APP_FILEMANAGEMENT_API_NAMESPACE=${API_URL}/assets
-REACT_APP_JOBMANAGER_API_NAMESPACE=${API_URL}/jobs
-REACT_APP_PIPELINES_API_NAMESPACE=${API_URL}/pipelines
-REACT_APP_CATEGORIES_API_NAMESPACE=${API_URL}/annotation
-REACT_APP_TOKENS_API_NAMESPACE=${API_URL}/processing
-REACT_APP_AUTH_API_NAMESPACE=${API_URL}/auth
-REACT_APP_USERS_API_NAMESPACE=${API_URL}/users
-REACT_APP_MODELS_API_NAMESPACE=${API_URL}/models
-REACT_APP_SEARCH_API_NAMESPACE=${API_URL}/search
-REACT_APP_TAXONOMIES_API_NAMESPACE=${API_URL}/taxonomy
+REACT_APP_FILEMANAGEMENT_API_NAMESPACE=/api/v1/assets
```

Comment:
Did you check your local changes with local BE, not remote? Because locally it doesn't work